src/axiosWrappers.ts

```diff
@@ -6,15 +6,25 @@ import axios, {
   AxiosRequestConfig,
   AxiosResponse,
 } from 'axios';
+import * as packageJson from '../package.json';
 import { handleErrorResponse } from './error';
 import { makeRequestSafeToRetry } from './retryRequests';
 import { transformDateInRequest, transformDateInResponse } from './utils';
 
 /** @hidden */
-export function generateAxiosInstance(baseUrl: string): AxiosInstance {
+export function generateAxiosInstance(
+  baseUrl: string,
+  appId?: string
+): AxiosInstance {
+  const headers: { [key: string]: string } = {
+    'x-cdp-sdk': `CogniteJavaScriptSDK:${packageJson.version}`,
+  };
+  if (appId) {
+    headers['x-cdp-app'] = appId;
```

Comment:
isn't this one has to be called `x-cdp-application`?

Reply:
It should be `x-cdp-app`